test-bench.sh

```diff
@@ -159,6 +160,9 @@ while [ $# -gt 0 ]; do
       ;;
     --jvm-args)
       JVM_ARGS_INPUT="$2"
+      ;;
+	--file-idf)
```

Comment:
do we have some help or text explaining all the options in the script (say on -h) ?

Reply:
Yes, typically we can access the helper text with the '--help' parameter.
Example: ./test-bench.sh run keygen --help